### ISSUE TITLE
Documentation ???

### ISSUE THREAD

**Issue by martialt** [centreon/centreon-sdk-python#14]
Hello,

There is a documentation somewhere ?

Regards,
Martial

**Comment by guillaumewatteeux** [comment]
Hi @martialt 

I just start it (https://pypi.org/project/centreonapi/) 
https://github.com/guillaumewatteeux/centreon-sdk-python/blob/dev/README.rst

I'll see to create a readthedocs.io (or other) online documentation.

**Comment by martialt** [comment]
Hi Guillaume,
Ok thanks for the doc.
* There is a typo issue. Missing double quotes after .tld
`centreon = centreon("https://centreon.mydomain.tld, "admin", "centreon")`
* And I tried to create a object "centreon" but I got an error. 
`Traceback (most recent call last):
  File "./myScript.py", line 14, in <module>
    centreon = centreon("http://hostname.com", "admin", "centreon")
NameError: name 'centreon' is not defined`

My code
`from centreonapi.centreon import Webservice`
`centreon = centreon("http://hostname.com", "admin", "centreon")`

Thanks
